torax/_src/imas_tools/input/core_profiles.py

```
@@ -109,3 +110,110 @@ def profile_conditions_from_IMAS(
       "normalize_n_e_to_nbar": False,
       "v_loop_lcfs": v_loop_lcfs,
   }
+
+
+def plasma_composition_from_imas(
+    ids: ids_toplevel.IDSToplevel,
+    t_initial: float | None = None,
+) -> Mapping[str, Any]:
+  """Returns dict with args for plasma_composition config from a given ids.
+
+  Loading IMAS data for plasma composition should only be used with n_e_ratios
+  and n_e_ratios_Z_eff impurity modes, not with fractions mode. The impurity
+  mode needs to be specified explicitly after loading the IMAS data. In case
+  n_e_ratios_Z_eff is specified, one impurity ratio must be set to None
+  explicitly. In case n_e_ratios is used, Z_eff will simply be ignored.
+  Note that if the ids indivual ions properties are not filled, it will not
+  raise an error and just return an empty dict as main_ion and species.
```

should we raise an error here to fail loudly and informatively?